opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -560,6 +560,16 @@ opennms.newsFeedPanel.show=true
 #
 #opennms.newsFeedPanel.url=https://www.opennms.com/feed/
 
+# This value enables or disables display of the Zenith Connect which allows
```

Comment:
We stopped putting any new properties in opennms.properties file because it results in upgrade issues.

Defaults can be in the code. Any overrides needs to be in opennms.properties.d/ at run time.

Reply:
Redirect back into Meridian still doesn't work. There are issues with retaining the query string past login, and also redirect seems to log the user out.

After registration, the user will get a Zenith screen where they can copy the token. That screen also has a link back to Meridian which the user could paste into the location bar of an already active Meridian session, which will go to the Meridian `zenith-connect-success` endpoint which could be used to save the token.

Reply:
I removed the changes from `opennms.properties` file. I will open a separate issue to add documentation, and also to fix an issue where Connect to Zenith is still showing up in the non-Vue menu.